Bela/main.cpp

```diff
@@ -245,24 +314,40 @@ bool csound_setup(BelaContext *context, void *p)
   csound->SetExternalMidiOutCloseCallback(CloseMidiOutDevice);
   /* set up digi opcodes */
   if(csnd::plugin<DigiIn>((csnd::Csound *) csound->GetCsound(), "digiInBela",
-			  "k","i", csnd::thread::ik) != 0)
+        "k","i", csnd::thread::ik) != 0)
     printf("Warning: could not add digiInBela k-rate opcode\n");
   if(csnd::plugin<DigiIn>((csnd::Csound *) csound->GetCsound(), "digiInBela",
-			  "a", "i", csnd::thread::ia) != 0)
+        "a", "i", csnd::thread::ia) != 0)
     printf("Warning: could not add digiInBela a-rate opcode\n");
   if(csnd::plugin<DigiOut>((csnd::Csound *) csound->GetCsound(), "digiOutBela" ,
-			   "", "ki", csnd::thread::ik) != 0)
+         "", "ki", csnd::thread::ik) != 0)
     printf("Warning: could not add digiOutBela k-rate opcode\n");
   if(csnd::plugin<DigiOut>((csnd::Csound *) csound->GetCsound(), "digiOutBela" ,
-			   "", "ai", csnd::thread::ia) != 0)
+         "", "ai", csnd::thread::ia) != 0)
     printf("Warning: could not add digiOutBela a-rate opcode\n");
   if(csnd::plugin<DigiIO>((csnd::Csound *) csound->GetCsound(), "digiIOBela" ,
-			   "", "kkk", csnd::thread::ik) != 0)
+         "", "kkk", csnd::thread::ik) != 0)
     printf("Warning: could not add digiIOBela k-rate opcode\n");
   if(csnd::plugin<DigiIO>((csnd::Csound *) csound->GetCsound(), "digiIOBela" ,
-			   "", "aaa", csnd::thread::ia) != 0)
+         "", "aaa", csnd::thread::ia) != 0)
     printf("Warning: could not add digiIOBela a-rate opcode\n");
-  
+
+  /* set up trill opcode */
+  if(csnd::plugin<TrillIn>((csnd::Csound *) csound->GetCsound(), "trill" ,
+         "kk[]k[]k[]", "ii", csnd::thread::ik) != 0)
+    printf("Warning: could not add trillBar a-rate opcode\n");  
+
+  /* set up trill sensors */
+  unsigned int i2cBus = 1;
+  for(uint8_t addr = 0x20; addr <= 0x50; ++addr) {
+    Trill::Device device = Trill::probe(i2cBus, addr);
+    if(Trill::NONE != device && Trill::CRAFT != device) {
```

Comment:
remove ` && Trill::CRAFT != device`, otherwise it will only work with Trill Craft

Reply:
as long as it's not `NONE`, you are good to go!

Reply:
Done